Library/ViewModels/ThanksViewModel.swift

```diff
@@ -86,11 +86,25 @@ public final class ThanksViewModel: ThanksViewModelType, ThanksViewModelInputs,
 
     self.backedProjectText = self.configureWithDataProperty.signal
       .skipNil()
-      .map { project, _, _, _ in
-
-        let string = Strings.You_have_successfully_backed_project_html(
-          project_name: project.name
-        )
+      .map { project, _, _, pledgeTotal in
+
+        let string: String
+
+        if featurePostCampaignPledgeEnabled(),
+          project.isInPostCampaignPledgingPhase,
+          let email = AppEnvironment.current.currentUserEmail {
+          let formattedTotal = Format.formattedCurrency(pledgeTotal, country: project.country)
```

Comment:
@scottkicks This is my first foray into formatting monetary strings in the app. Am I pulling the correct fields? `project.country` seemed like the more immediate/correct option, versus `project.stats.country` but let me know if I'm missing something.

Reply:
As a note for future refactoring, we should really be passing around something like `Money` type that includes the currency with the amount. The raw double being passed around makes me nervous.

Reply:
Yes, this is how currency formatting is being handled elsewhere in the checkout flow. I agree it's not the best. I'm pushing to get some time allocated after the first release of PCP to make refactors like this. It's a little too in the weeds right now, I think.